etc/Env.sh

```diff
@@ -2,6 +2,9 @@
 
 set -euo pipefail
 
+# Make sure we are on the correct folder before beginning
+cd "$(dirname $(readlink -f $0))"
```

Comment:
```suggestion
cd "$(dirname $(readlink -f $0))/../"
```

Sorry I jumped the gun on the copy-pasta